pkg/ruleengine/v1/helpers/iouring/iouring.go

```diff
@@ -0,0 +1,138 @@
+package iouring
+
+const (
```

Comment:
Use this: https://github.com/Iceber/iouring-go/blob/002cfd2e2a9021c13192f3bd4d9023374e38c93f/syscall/types.go#L27